### PR TITLE
Allow datastore to operate in read-only mode

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/1158/

File: AppController/djinn.rb [hunk 1]
```diff
@@ -1366,6 +1370,48 @@ def set_deployment_id(secret, id)
...
+  # Enables or disables datastore writes on this node.
+  # Args:
+  #   read_only: A string that indicates whether to turn read-only mode on or
+  #     off.
+  def set_node_read_only(read_only, secret)
+    if !valid_secret?(secret)
+      return BAD_SECRET_MSG
+    end
+
+    if read_only == 'true'
```
Comment (menivaitsi) [inline AppController/djinn.rb]: Can we pass in the boolean straight up instead of a string?

**Comment by menivaitsi** [comment]
:+1:

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/1166/

File: scripts/read_only.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python2
```
Comment (menivaitsi) [inline scripts/read_only.py]: Please a docstring explaining what this script does.

**Comment by menivaitsi** [comment]
Please run another build since the branch has been updated.

**Pull request merged**